Sources/SnapshotTesting/Common/View.swift

```diff
@@ -852,7 +852,11 @@ extension View {
                 callback(Image())
                 return
               }
-              wkWebView.takeSnapshot(with: nil) { image, _ in
+              let configuaration = WKSnapshotConfiguration()
+              if #available(iOS 13, *) {
```

Comment:
It might be nice to add macOS 10.15 here too since it also supports `afterScreenUpdates` and could be using this renderer.

Reply:
Fixed!